.gitmodules

```diff
@@ -0,0 +1,73 @@
+# This is a git-fleximod adapted .gitmodules file.  Any field with a name starting in fx is a git-fleximod
+# specific field.  See https://github.com/ESMCI/git-fleximod for details.
+
+[submodule "cism"]
```

Comment:
Is there any problem with having a submodule named "cism" inside of what will probably be a submodule named "cism" in CESM? Maybe this should be "cism-standalone" or "cism-source"?

Reply:
Good point.  I'll change this.